cli/admin/src/main/java/io/pravega/cli/admin/AdminCommand.java

```diff
@@ -390,11 +390,21 @@ public SegmentHelper instantiateSegmentHelper(CuratorFramework zkClient) {
                 .containerCount(getServiceConfig().getContainerCount())
                 .build();
         HostControllerStore hostStore = HostStoreFactory.createStore(hostMonitorConfig, StoreClientFactory.createZKStoreClient(zkClient));
-        ClientConfig clientConfig = ClientConfig.builder()
-                .controllerURI(URI.create(getCLIControllerConfig().getControllerGrpcURI()))
-                .validateHostName(getCLIControllerConfig().isAuthEnabled())
-                .credentials(new DefaultCredentials(getCLIControllerConfig().getPassword(), getCLIControllerConfig().getUserName()))
-                .build();
+
+        ClientConfig.ClientConfigBuilder clientConfigBuilder = ClientConfig.builder()
```

Comment:
The previous approach for building the configuration seems more concise. Please, try to keep it similar to what it was if possible, while setting the right values.

Reply:
I dont think I can keep it the same, I am doing this approach based on other examples. I dont think we should be passing in some of those values into ClientConfig if tls or auth isnt enabled.